src/Makefile.am

```diff
@@ -675,11 +675,17 @@ repl_modules.cc: repl_modules.sh Makefile
 
 include $(top_srcdir)/doc/manuals/Substitute.am
 
-squid.8: $(srcdir)/squid.8.in Makefile
-	$(SUBSTITUTE) < $(srcdir)/squid.8.in > $@
+squid.md: $(srcdir)/squid.md.in Makefile
```

Comment:
What Markdown syntax variant do you propose to standardize on? Pandoc claims to support several [variants](https://pandoc.org/chunkedhtml-demo/8.22-markdown-variants.html), so using pandoc does not, by itself, define the flavor we should be using. Now would be the right time to specify it!

Reply:
https://commonmark.org/help/